src/main/java/com/epam/ta/reportportal/core/configs/ExecutorConfiguration.java

```diff
@@ -54,7 +54,7 @@ public TaskExecutor saveLogsTaskExecutor() {
 		ThreadPoolTaskExecutor executor = new ThreadPoolTaskExecutor();
 		executor.setCorePoolSize(20);
 		executor.setMaxPoolSize(100);
-		executor.setQueueCapacity(600);
+		executor.setQueueCapacity(5000);
```

Comment:
does it make sense to move it to properties?

Reply:
moved to the application.yaml